src/main/java/io/codelex/oop/runners/Runner.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.oop.runners;
+
+public enum Runner {
+    BEGINNER(300,420),
+    INTERMEDIATE(210, 299),
+    ADVANCED(120, 209);
+
+    final int minimumTime;
```

Comment:
private

Reply:
IntelliJ gan saka, ka private ir redundant, respektīvi, vērtības automātiski ir private tipa. Vai tad tā nav?

Reply:
jāizveido tad arī getter metode, lai varētu peikļaut laukam ārpuss enum, šeit vnk tev tas nav nepieciešams, btu in real life it always will be like that

un vaiable acces ir pilnība atkarīgs no tevis, šeti tev pašlaik ir `protected`

Reply:
Paldies!